.github/workflows/terraform-observe_pre-commit.yaml

```diff
@@ -124,6 +124,7 @@ jobs:
         name: Setup TFLint
         with:
           tflint_version: ${{ inputs.tflint-version }}
+          github_token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
This already happens by default:

https://github.com/terraform-linters/setup-tflint/blob/22b85f6a023e77d0c3a01f27820f353968ade275/action.yml#L11

This is not the step that's failing, it's the next one.

Reply:
See https://github.com/terraform-linters/setup-tflint#usage for how to pass via env so it's exposed when calling `tflint` itself